plugins/services/src/js/structs/Service.js

```diff
@@ -7,6 +7,10 @@ import ServiceSpec from "./ServiceSpec";
 import VolumeList from "./VolumeList";
 
 module.exports = class Service extends Item {
+  constructor() {
+    super(...arguments);
+    this._regions = undefined;
```

Comment:
why `undefined` and not `null`?

Reply:
why `null` and not `undefined`? :trollface: 

scnr 😅  - havent thought about it when writing the code 🤷‍♂️ I'm not aware that we use one of them consistently thorough our codebase. Do we want to do so?

Reply:
thinking about it now, an empty array would have been better to be typesafe 🤔 will change 👍